files/ko/web/api/clipboard_api/index.md

```diff
@@ -38,15 +38,15 @@ navigator.clipboard.readText().then(
 
 ### Clipboard
 
-{{Compat("api.Clipboard")}}
+{{Compat}}
 
 ### ClipboardEvent
 
-{{Compat("api.ClipboardEvent")}}
+{{Compat}}
 
 ### ClipboardItem
 
-{{Compat("api.ClipboardItem")}}
+{{Compat}}
```

Comment:
A little after-merge review: sections like this with multiple interfaces should actually just be replaced with a single `{{Compat}}` call -- the macro (well, "macro") has been enhanced to do this for us if multiple BCD keys are given in the frontmatter keys!

Reply:
What a nice catch! Why didn't I think there would be more than one Compat on a page? I think it would be better to just update the entire page. Thanks again😸

Reply:
There's a lot of little details like this left over from the old wiki days, and it's hard to catch them all -- I had just found out that some `{{Specifications}}` macro calls can also have arguments that need to be removed, apparently, and I hadn't even thought about that until stumbling across it in the Spanish locale!